scripts/export/project_builder.py

```diff
@@ -0,0 +1,665 @@
+        )
+        logger.info(
+            'Created %d individual external tool licence files.',
+            len(external_files_created),
+        )
+
+        # 3. --- Generate extra licence information notices ---
+
+        notice_file = self.licences_dir / 'NOTICE.txt'
+        logger.info('Creating notice file [%s]...', notice_file)
+
+        with open(notice_file, 'w', encoding='utf-8') as f:
+            f.write(f"""SHARLY CHESS {SHARLY_CHESS_VERSION}
+    {SharlyChessConfig.en_copyright}
+    {SharlyChessConfig.url}
+
+    This software includes third-party libraries and components.
+    See THIRD_PARTY_LICENSES.txt for detailed license information.
+
+    For a summary of all licenses used, see LICENSE_SUMMARY.md.
+    Individual package licenses are available in the packages/ subdirectory.
+
+    """)
+
+            # Add info about major license types that require attribution
+            f.write("""IMPORTANT LICENSE NOTICES:
+
+    1. GNU LGPL Components:
+       Some components are licensed under GNU Lesser General Public License (LGPL).
+       Source code for these components is available from their respective PyPI packages.
+
+    2. Apache License Components:
+       Some components are licensed under Apache License 2.0.
+       See THIRD_PARTY_LICENSES.txt for required copyright notices.
+
+    3. BSD License Components:
+       Some components are licensed under various BSD licenses.
+       See THIRD_PARTY_LICENSES.txt for required copyright notices.
+
+    4. Mozilla Public License Components:
+       Some components are licensed under Mozilla Public License 2.0.
+       See THIRD_PARTY_LICENSES.txt for complete license terms.
+
+    For complete license terms and copyright notices, please refer to
+    THIRD_PARTY_LICENSES.txt or the individual package files in packages/.
+    """)
+
+        logger.info('Licence files generated successfully.')
+        return True
+
+    def _build_exe(self) -> bool:
+        pyinstaller_params = [
+            '--clean',
+            '--noconfirm',
+            '--name=' + self.basename,
+            '--copy-metadata',
+            'sharly_chess',
+            '--hiddenimport=common',
+            '--hiddenimport=data',
+            '--hiddenimport=database',
+            '--hiddenimport=pairing',
+            '--hiddenimport=plugins',
+            '--hiddenimport=web',
+            '--hiddenimport=babel.numbers',
+            '--hiddenimport=pyexcel_io.writers',
+            '--hiddenimport=colorlog',
+            '--paths=.',
+            f'--icon=src/web/static/images/{self.project_name}.ico',
+            '--optimize',
+            '1',
+            # TODO Remove this option when https://github.com/pyinstaller/pyinstaller/issues/9149 is fixed
+            # this option was added in 2.7.2 as a workaround of a bug in PyInstaller
+            # See https://github.com/pyinstaller/pyinstaller/issues/9149#issuecomment-2914294505
+            '--exclude-module=pkg_resources',
```

Comment:
Pyinstaller 6.14.0 removed the `packaging` hook: https://pyinstaller.org/en/v6.15.0/CHANGES.html#id8

You might want to update our export dependencies to use `PyInstaller >= 6.14.0`